### PR TITLE
Feature/#87 partners list api

### PR DESCRIPTION
**Pull request by kenzo23312**
Api list integration

### CODE REVIEW

File: frontend/src/components/common/list/paginatedList.js [hunk 1]
```diff
@@ -18,66 +22,155 @@ const styleSheet = createStyleSheet('paginatedList', (theme) => {
...
+  updatePageNumber(pageNumber) {
+    const { pathName, query } = this.props;
+
+    history.push({
+      pathname: pathName,
+      query: R.merge(query, { page: pageNumber + 1 }),
+    });
+  }
+
+  listLoader() {
```
Comment (marcindo) [inline frontend/src/components/common/list/paginatedList.js]: I implemented list loader for selectable list, can you merge both functionalities?

File: frontend/src/components/common/list/paginatedList.js [hunk 2]
```diff
@@ -18,66 +22,155 @@ const styleSheet = createStyleSheet('paginatedList', (theme) => {
...
+      <div style={{ position: 'relative' }}>
+        <Grid
+          rows={items}
+          columns={columns}
+          headerPlaceholderTemplate={() => this.navigationHeader()}
+        >
+          <PagingState
+            currentPage={pageNumber - 1}
+            pageSize={pageSize}
+            onPageSizeChange={(size) => { this.onPageSize(size); }}
```
Comment (marcindo) [inline frontend/src/components/common/list/paginatedList.js]: can you move those callbacks to separate file and add it to selectable list too?

File: frontend/src/components/partners/partnersContainer.js [hunk 3]
```diff
@@ -1,46 +1,60 @@
...
 
 const messages = {
   header: 'Partners',
 };
 
-class PartnersContainer extends Component {
-  onRowClick(row) {
-    history.push('/partner/1/overview');
+class PartnersContainer extends Component { 
+  shouldComponentUpdate(nextProps, nextState) {
```
Comment (marcindo) [inline frontend/src/components/partners/partnersContainer.js]: I would like to have this functionality moved to some utils as it is pretty generic, it should be a function which gets the call back for api action as parameter

File: frontend/src/components/partners/partnersContainer.js [hunk 4]
```diff
@@ -1,46 +1,60 @@
...
+    return true;
   }
 
-  static partnerDetailCell(row) {
-    return (
-      <PartnerProfileDetailItem partner={row.details} />
-    );
+  /* eslint-disable class-methods-use-this */
+  onRowClick(row) {
+    history.push(`/partner/${row.id}/overview`);
```
Comment (marcindo) [inline frontend/src/components/partners/partnersContainer.js]: where is onRowClick used?

Comment (kenzo23312) [reply frontend/src/components/partners/partnersContainer.js]: done

File: frontend/src/components/partners/partnersContainer.js (hunk 3 above)
Comment (kenzo23312) [reply frontend/src/components/partners/partnersContainer.js]: done

File: frontend/src/components/common/list/paginatedList.js (hunk 1 above)
Comment (kenzo23312) [reply frontend/src/components/common/list/paginatedList.js]: done

File: frontend/src/components/common/list/paginatedList.js (hunk 2 above)
Comment (kenzo23312) [reply frontend/src/components/common/list/paginatedList.js]: done

**Pull request merged**